README.md

```diff
@@ -33,7 +33,16 @@ npm install
 
 ## Usage
 
-TBD
+```js
+import {getRemoteStorage} from 'bedrock-web-data-hub';
+import VerifiableCredentialStore from 'bedrock-web-vc-store';
+
+const remoteStorage = await getRemoteStorage({accountId: 'test'});
+
+const vcStore = new VerifiableCredentialStore({accountId: 'test', remoteStorage});
```

Comment:
@dlongley Is this the right idea, in terms of creating a `VerifiableCredentialStore` instance?

Does the upstream code have to set up a `remoteStorage`, and pass it into the VC store?
Is the VC store dependent on a particular `accountId`?

Reply:
@dmitrizagidulin, I recommend upstream passes in a RemoteStorage API for now -- and you don't concern yourself with "accounts" at all. We'll see if that flies.